client/components/App.js

```diff
@@ -1,17 +1,23 @@
 var React = require('react');
+var { connect } = require('react-redux');
+
 var styles = require('./App.css');
 
-var App = React.createClass({
+var App = connect(state => ({count: state}))(React.createClass({
```

Comment:
Oh, state is actually just `{num: 5}`. if you don't name it then the prop would have `this.props.num` .

As for why I renamed state to count, it was because when I first started looking at the project I kept getting this.props.state confused with react's this.state. So I renamed it to make it more clear to myself.